src/test/groovy/graphql/schema/GraphQLAppliedDirectiveArgumentTest.groovy

```diff
@@ -1,16 +1,16 @@
-package graphql
+package graphql.schema
 
+import graphql.GraphQLContext
+import graphql.TestUtil
 import graphql.execution.ValuesResolver
-import graphql.schema.GraphQLArgument
 import graphql.schema.idl.RuntimeWiring
 import graphql.schema.idl.SchemaGenerator
 import graphql.schema.idl.SchemaParser
 import graphql.schema.idl.errors.SchemaProblem
 import spock.lang.Specification
 
-class Issue2001 extends Specification {
-
-    def "test non-list value for a list argument of a directive"() {
+class GraphQLAppliedDirectiveArgumentTest extends Specification {
+    def "test non-list value for a list argument of a directive - issue 2001"() {
```

Comment:
I've kept the "issue 2001" in the test name in case someone wants to refer back to it